### ISSUE TITLE
Missing images in "Your XXX Teams" email

### ISSUE THREAD

**Issue by stephenfin** [mattermost/platform#957]
**NOTE:** These issues were seen on v1.0.0 of Mattermost. They may already be fixed but I haven't figured out how to build the newer versions from source yet (I'm new to Golang, and ReactJS for that matter).

When I enter my email to do a "Find my team" search, I successfully receive an email listing the teams that I'm a member of. However, there are two issues with the images in this email and both revolve around the `src` URL of the images:

```html
<td style="padding: 20px 20px 10px; text-align:left;">
    <img src="/static/images/Mattermost%20Test-logodark.png" width="130px" style="opacity: 0.5" alt="">
</td>
```

```html
<p style="margin: 25px 0;">
    <img width="65" src="/static/images/circles.png" alt="">
</p>
```

Firstly, the URL is a relative URL. This should be easy to fix as absolute URLs are used later in the mail:

```html
Your request to find teams associated with your email found the following:<br>
                                                        
    <a href="http://mattermost.example.com/my-team" style="text-decoration: none; color:#2389D7;">my-team</a><br>
```

Secondly, in the case of the `[TEAM_NAME]-logodark.png`, the image in question doesn't actually exist and there appears to be no way to add it (besides manually adding the file on the server). This would hint at two changes:
<ol type="a">
<li>An option should be added to the web UI to upload team logo images for the server (perhaps as part of the `SETTINGS` > `Service Settings` pane)</li>
<li>Perhaps the images should be stored in the `Local Directory Location` (`SETTINGS` > `File Settings`) which is writeable, while `$INSTALL_DIR/static/images` really should not be.</li>
</ol>

PS: One other (minor) usability improvement: change "Find my team" to "Find my teams"

**Comment by coreyhulen** [comment]
Fixed with PR https://github.com/mattermost/platform/pull/970

Issues address:
1. Relative links fixed
2. The logo is now just `.../static/images/logo-email.png`.  I've created https://mattermost.atlassian.net/browse/PLT-566 to track the issue of uploading a new logo.  Since the original code would break for most people the recommendation is to replace logo-email.png with your logo until PLT-566 is implemented.
3. "Find my team" -> "Find my teams"
